plugins/net_plugin/net_plugin.cpp

```diff
@@ -3518,9 +3521,11 @@ namespace eosio {
                controller& cc = my_impl->chain_plug->chain();
                std::optional<block_id_type> peer_lib_id = cc.fork_block_id_for_num( peer_lib ); // thread-safe
                if (!peer_lib_id) {
+                  // can be not found if running with a truncated block log
                   peer_dlog( this, "peer last irreversible block ${pl} is unknown", ("pl", peer_lib) );
```

Comment:
Should this be an info level log?

Reply:
I don't think so, if running without a block log you would get that on every connection.